### PR TITLE
Add resque background job adapter

### PR DESCRIPTION
**Pull request by dsalahutdinov**

### CODE REVIEW

**Comment by palkan** [comment]
Cool! 👍 

Let's update Readme and add a changelog entry.

**Comment by dsalahutdinov** [comment]
it's done

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 ## master
 
+## 0.2.0 (2018-02-22)
+
+ - [PR [#8](https://github.com/palkan/isolator/pull/8)] Add resque background job adapter.
```
Comment (palkan) [inline CHANGELOG.md]: What about adding your nickname?)

**Comment by dsalahutdinov** [comment]
I've added 😄

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 ## master
 
+## 0.2.0 (2018-02-22)
+
+ - [PR [#8](https://github.com/palkan/isolator/pull/8)] Add resque background job adapter. ([@dsalahutdinov][])
```
Comment (palkan) [inline CHANGELOG.md]: You should add the link itself to the end of file. It's just a reference

**Comment by dsalahutdinov** [comment]
done, sorry for stupidity 😸

**Comment by palkan** [comment]
Thanks! Released as 0.2.0

**Pull request merged**